core/src/main/java/org/testcontainers/images/builder/ImageFromDockerfile.java

```diff
@@ -128,9 +132,15 @@ public void onNext(BuildResponseItem item) {
                     Transferable transferable = entry.getValue();
                     final String destination = entry.getKey();
                     transferable.transferTo(tarArchive, destination);
+                    bytesToDockerDaemon += transferable.getSize();
                 }
                 tarArchive.finish();
             }
+            
+            log.info("Transferred {} KB to Docker daemon", bytesToDockerDaemon % 1024);
```

Comment:
Are you sure about this calculation?

Shouldn't it be:
```suggestion
            log.info("Transferred {} KB to Docker daemon", bytesToDockerDaemon / 1024);
```

Reply:
Also, why not use a consistent unit (either KB or MB) for both log messages?

Reply:
I wanted to use MB for the "woa you transferred a lot of data" message, because at that order of magnitude it will be more easily readable. However, for this log statement there will be a lot of cases where <1MB will be transferred and I didn't want to report "0 MB of data transferred" because that's confusing. Also didn't think it was worth branching the code to do "report in KB if <1MB otherwise report in MB"

Reply:
We happen to have commons-io `FileUtils` on the classpath - we could actually delegate all this formatting to its `byteCountToDisplaySize` method, and not have to worry about which is the most appropriate unit for either message. WDYT?